doc/OnlineDocs/explanation/modeling/dae.rst

```diff
@@ -416,7 +416,7 @@ discretization equations for this method are shown below:
 
 where :math:`h` is the step size between discretization points or the size of
 each finite element. These equations are generated automatically as
-:py:class:`Constraints<.Constraint>` when the backward
+:py:class:`.Constraint` when the backward
```

Comment:
Could you revert this one back to :py:class:`Constraints <.Constraint>`? I wanted to use the plural of the class name here.